runtests.py

```diff
@@ -57,6 +57,14 @@ def djconnectwise_configuration():
             'LOCATION': 'unique-snowflake',
         }
     },
+    LOGGING={
```

Comment:
Thought I'd make the test output slightly quieter.

Reply:
Yeah, much appreciated. I kept meaning to do this myself.